transport/cluster.go

```diff
@@ -3,42 +3,113 @@ package transport
 import (
 	"fmt"
 	"log"
+	"strconv"
 	"time"
 
 	"scylla-go-driver/frame"
 	. "scylla-go-driver/frame/response"
 
+	"github.com/google/btree"
 	"go.uber.org/atomic"
 )
 
 type (
-	PeerMap = map[string]*Node
+	PeerMap         = map[string]*Node
+	DCRacksMap      = map[string]int
+	DCRepFactorsMap = map[string]repFactor
+	KSMap           = map[string]keyspace
 
 	requestChan chan struct{}
+
+	repFactor uint32
```

Comment:
Often referred as `rf`, that is to say dcRfMap looks better. 
Not sure we need a type for that.
This is an int a small one usually `3`, what is the benefit of the dedicated type?